assembly/bindgen.ts

```diff
@@ -14,6 +14,12 @@ function isNull<T>(t: T): bool {
   return false;
 }
 
+//@ts-ignore
+@global
+function notPayable(): void {
```

Comment:
Why make it global instead of an export?

Reply:
Because it will be injected into the source once we have `@payable` support.